src/utils/script_args.py

```diff
@@ -64,6 +56,7 @@ def generic_script_init(description: str) -> ScriptArgs:
         "Primarily intended for deployment in staging environment.",
         default=False,
     )
+    # TODO: this should be per token (like list[tuple[Token,minAmount]])
```

Comment:
I would reference to a ticket for this. What do you think?

Reply:
Good thinking. Didn't want to change it here because it also requires changing deployments.

Reply:
Issue here: https://github.com/cowprotocol/solver-rewards/issues/314